src/react-components/src/components/Link/LinkTypes.tsx

```diff
@@ -0,0 +1,12 @@
+export enum LinkTypes {
+    Action = "action",
+    Backwards = "backwards",
+    Button = "button",
+    Default = "default",
+    Forwards = "forwards",
+}
+
+export enum LinkIconPositions {
```

Comment:
considering that `LinkIconPositions` is only used internally as a constant in `Link` and never passed in, I think it doesn't need to be here --   It can just be assigned as string - in `Link.tsxL63` and `L66` There's no need to do validation against a set of allowed values because it's just an internal variable used internally.